src/main/java/ludum/dare/components/HurtComponent.java

```diff
@@ -0,0 +1,51 @@
+package ludum.dare.components;
+
+import com.badlogic.gdx.math.Vector2;
+import ludum.dare.components.upgradeComponents.IRemoveable;
+import ludum.dare.components.upgradeComponents.TimedComponent;
+import ludum.dare.interfaces.IComponent;
+import ludum.dare.interfaces.IUpdate;
+
+public class HurtComponent implements IComponent, IUpdate, IRemoveable {
+
+    private InputComponent input;
+    private TimedComponent timer;
+    private boolean shouldRemove = false;
+
+    public HurtComponent(InputComponent input, float timeLimit) {
+        timer = new TimedComponent(timeLimit);
+        this.input = input;
+        if (input != null) input.inControl(false);
+    }
+
+    public HurtComponent(InputComponent input, float timeLimit, PhysicsComponent phys, float force, Vector2 direction) {
+        this(input, timeLimit);
+        if (phys != null) {
+            // TODO: how to hell do you push a body around?  I want to implement knockback
```

Comment:
I wanted to try to bump the physics body around, like send it flying when it gets hit, but I couldn't figure it out